src/components/Dropdowns/ColorRampDropdown.tsx

```diff
@@ -17,6 +17,9 @@ import IconButton from "../IconButton";
 
 import styles from "./ColorRampDropdown.module.css";
 
+const COLOR_RAMP_BUTTON_CLASS = "color-ramp-button";
+const COLOR_PALETTE_BUTTON_CLASS = "color-palette-button";
+
```

Comment:
Adds classes to the ramp and palette buttons so they can be used to filter click events in GA.

Other changes:
- Added missing aria-label tags
- Add key of each color ramp/palette as the ID of the button elements